input/groups/armor.json

```diff
@@ -1603,26 +1557,24 @@
         9316,
         9327
       ],
-      "Standalone: Grothmar Valley": [
+      "Standalone: Strike Mission": [
```

Comment:
It's supposed to cover all standalone strike mission collectibles of the parent category, so going after each specific strike mission would require multiple groups, it would be a mess.

This one just happens to have one single entry, but the general rule is the same.